cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -137,7 +137,7 @@ endfunction()
 # ----------------------------------------------------------------------
 # ORC Format
 ExternalProject_Add (orc-format_ep
-  URL "https://dlcdn.apache.org/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz"
+  URL "https://www.apache.org/dyn/closer.lua/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz?action=download"
   URL "https://archive.apache.org/dist/orc/orc-format-${ORC_FORMAT_VERSION}/orc-format-${ORC_FORMAT_VERSION}.tar.gz"
```

Comment:
We can remove this archive.apache.org URL. The `closer.lua` URL redirects to the archive.apache.org URL automatically when the target file doesn't exist in https://dist.apache.org/repos/dist/release/orc/ .

Reply:
Sure, let me clean up this too.